packages/relay/src/types/Report.ts

```diff
@@ -0,0 +1,32 @@
+export enum ReportCategory {
+    Attack = 0, // 對使用者、特定個人、組織或群體發表中傷、歧視、挑釁、羞辱、謾罵、不雅字詞或人身攻擊等言論
+    Spam = 1, // 張貼商業廣告內容與連結、邀請碼或內含個人代碼的邀請連結等
+    R18 = 2, // 張貼色情裸露、性暗示意味濃厚的內容，惟內容具教育性者不在此限
+}
+
+export enum ReportType {
+    Post = 0,
+    Comment = 1,
+}
+
+export interface ReportHistory {
+    reportId?: string
+    type: number // 0: Post, 1: Comment
+    objectId: string // PostId or CommentId
+    reportorEpochKey: string // Epoch Key of the person who reported
+    reportorClaimedRep?: boolean // TRUE: claimed, FALSE: not claimed
+    respondentEpochKey?: string // Epoch Key of the person who was reported
+    respondentClaimedRep?: boolean // TRUE: claimed, FALSE: not claimed
+    reason: string // Reason of the report
+    adjudicateCount?: number // Number of voters
+    adjudicatorsNullifier?: {
```

Comment:
this is an Object type, https://github.com/vimwitch/anondb/blob/ed82c0a7a11a315344f19c0d79ddfb5f6edf5467/examples/schema.types.ts#L65

Reply:
In my understanding, this is compatible and have a better readability correct me if I'm wrong :)